cmd/meroxa/root/apps/deploy_test.go

```diff
@@ -1162,3 +1162,83 @@ func TestUploadFile(t *testing.T) {
 		})
 	}
 }
+
+func TestTeardown(t *testing.T) {
```

Comment:
Thank you for all the test coverage here as well, this is greatly appreciated! :sparkles: